jerry-ext/module/module.c

```diff
@@ -14,11 +14,13 @@
  */
 
 #include <string.h>
+#include "jmem.h"
```

Comment:
I think we should not use internal headers in `jerry-ext`. I don't see what are you using from it. Please remove it if possible.